openassessment/xblock/static/js/src/lms/oa_staff_info.js

```diff
@@ -53,12 +53,52 @@ OpenAssessment.StaffInfoView.prototype = {
             function(html) {
                 // Load the HTML and install event handlers
                 $('#openassessment__student-info', view.element).replaceWith(html);
+                view.ScoreOverrideHandler();
             }
         ).fail(function(errMsg) {
                 view.showLoadError('student_info');
             });
     },
+    
+    /**
+     Upon request, submits an override grade.
+     **/
+    peerScoreOverride: function() {
+    	var sel = $('#openassessment__staff-info', this.element);
+    	var student_id = sel.find('#openassessment__student_id').val();
+    	var points_override = sel.find('#openassessment_override_score').val();
+    	var points_possible = sel.find('#openassessment_points_possible').html();
+    	this.server.peerScoreOverride(student_id, points_override, points_possible).done(
+            function(points_override) {
+                $('#openassessment_points_override', this.element).html(points_override);
+            }
+        ).fail(function(errMsg) {
+                $('#openassessment_points_override', this.element).html(errMsg);
+        });
+    },
+    
+    /**
+     Handler for Peer score override.
+     This must be run after loadStudentInfo has completed so that
+     the required HTML elements are present.
+     **/
+    ScoreOverrideHandler: function() {
+        var sel = $('#openassessment__student-info', this.element);
+        var view = this;
 
+        if (sel.length <= 0) {
+            return;
+        }
+        
+        // Click handler for overriding a student's peer score.
+        sel.find('#submit_override_score').click(
```

Comment:
Given we're doing the lookup on ID, I wonder, how much we gain by `find`ing from a parent element instead of just querying by ID like `$('#submit_override_score')`.

Also, I'm assuming this all just breaks if you try to place more than one on a page?